frontend/src/metabase/ui/components/inputs/Input/Input.styled.tsx

```diff
@@ -28,10 +28,14 @@ export const getInputOverrides = (): MantineThemeOverride["components"] => ({
         minHeight: getSize({ size, sizes: SIZES }),
         background: theme.fn.themeColor("bg-white"),
         "&::placeholder": {
-          color: theme.fn.themeColor("text-light"),
+          color: "var(--mb-color-text-tertiary)",
```

Comment:
The color was broken before, neither value color nor placeholder color is visible, and they're set to `transparent`, I guess that the side effect from #37443.

Since Mantine V6's `TextInput`'s disabled state [still shows value color and placeholder color.
](https://v6.mantine.dev/core/text-input/)

![img](https://private-user-images.githubusercontent.com/1937582/363250046-20dc46ac-fd19-420b-9cb8-b63b19579c01.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUwMzU1MDUsIm5iZiI6MTcyNTAzNTIwNSwicGF0aCI6Ii8xOTM3NTgyLzM2MzI1MDA0Ni0yMGRjNDZhYy1mZDE5LTQyMGItOWNiOC1iNjNiMTk1NzljMDEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MzBUMTYyNjQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTY3ZWRlMTg0ZjgyYThmOWE4ZDA0NDBkODBmMDRhYWYxM2FkMDUxMjNjYTU0OWFhNDhmNTdmOWM2NGQxMWNhMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.VALMw2TMxmBxQLBycRQultdFMKZlRsVUrijbnvNm9xM)